db/migrate/20221109105200_change_sample_concentrations_to_single_value.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeSampleConcentrationsToSingleValue < ActiveRecord::Migration
+
+  def up
+    Sample.find_each do |sample|
+      if sample.core_fields.key?("concentration_number") && sample.core_fields.key?("concentration_exponent")
+        sample.core_fields["concentration"] = sample.core_fields["concentration_number"].to_i * (10**(sample.core_fields["concentration_exponent"].to_i))
```

Comment:
nitpick: we could improve readability by using variables

```suggestion
      number = sample.core_fields.key?("concentration_number").try(&:to_i)
      exponent = sample.core_fields.key?("concentration_exponent").try(&:to_i)

      if number && exponent
        sample.core_fields["concentration"] = number * (10**exponent)
```